src/resources/review/review.db.repository.js

```diff
@@ -0,0 +1,17 @@
+const Review = require('./review.model');
+
+const get = async () => {
+  const review = await Review.find();
+
+  return review;
+};
+
+const put = async arrReview => {
```

Comment:
as I understand `arr` prefix stands for array. In your case this is a single entity so you can use singular form